packages/template-retail-react-app/app/hooks/use-add-to-cart-modal.js

```diff
@@ -58,19 +57,12 @@ export const AddToCartModal = () => {
     if (!isOpen) {
         return null
     }
-    const {product, isProductPartOfSet, quantity} = data || {}
+    const {product, itemsAdded} = data || []
```

Comment:
Was this change intentional? My guess is that this was your intention:
```suggestion
    const {product, itemsAdded = []} = data || {}
```

Reply:
Yeah.. thats what I meant to doo .. duh